recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./configure --prefix=$PREFIX
+./configure --prefix=$PREFIX PERL='/usr/bin/env perl'
```

Comment:
This will be whatever perl is on PATH, not necessarily the system one.

If this fixes the problem, then I approve this PR.  Having the correct perl on PATH should be OK with any version of conda-build, but with the activation stuff, might work best in conda-build 2.